loading dataset
creating model
start training
torch.Size([128, 3]) dsd
Traceback (most recent call last):
  File "train_2D.py", line 71, in <module>
    loss.backward()
  File "/home/mmvc/.conda/envs/ys/lib/python3.6/site-packages/torch/tensor.py", line 93, in backward
    torch.autograd.backward(self, gradient, retain_graph, create_graph)
  File "/home/mmvc/.conda/envs/ys/lib/python3.6/site-packages/torch/autograd/__init__.py", line 90, in backward
    allow_unreachable=True)  # allow_unreachable flag
RuntimeError: one of the variables needed for gradient computation has been modified by an inplace operation
